src/chrome/content/rules/AddThis.xml

```diff
@@ -52,17 +41,15 @@
 	<target host="s7.addthis.com" />
 	<target host="s9.addthis.com" />
 	<target host="secure.addthis.com" />
-	<target host="status.addthis.com" />
 	<target host="su.addthis.com" />
 	<target host="support.addthis.com" />
-	<target host="www.addthis.com" />
 
 		<!--	Redirects to http:
 						-->
 
-		<!--exclusion pattern="^http://(?:www\.)?addthis\.com/(?:browser-extensions$|careers$|contact$|forgot-password$|plugins$|privacy/privacy-policy$|security/hall-of-fame$|social-buttons$|support$)" /-->
+		<!--exclusion pattern="^http://(www\.)?addthis\.com/(browser-extensions|careers|contact|forgot-password|plugins|privacy/privacy-policy|security/hall-of-fame|social-buttons|support)$" /-->
 
-		<exclusion pattern="^http://(?:www\.)?addthis\.com/(?!/*(?:$|\?|bookmark\.php|(?:academy|blog|dashboard|get|landing|login|plans|press|register|services|sitebar-editor|website-tools|wordpress)(?:$|[?/])|download/|images/|\w+/sharing-buttons(?:$|\?)|style/))" />
+		<exclusion pattern="^http://(www\.)?addthis\.com/(?!/*($|\?|bookmark\.php|(academy|blog|dashboard|get|landing|login|plans|press|register|services|sitebar-editor|website-tools|wordpress)($|[?/])|download/|images/|\w+/sharing-buttons($|\?)|style/))" />
```

Comment:
Could you double check this? I think we can get rid of the exclusions.

Reply:
All the negative test URLs redirect to HTTPS on their own now. I think the exclusion no longer applies.
Thanks for the heads up. Removing.